CMakeLists.txt

```diff
@@ -86,6 +86,7 @@ option(ENABLE_DOCS "Indicates whether documentation should be built." ON)
 option(ENABLE_TUTORIALS "Indicates whether tutorials should be build. Note that this also depends on the availability of (pdf)latex." ON)
 option(WITH_GUI "Build GUI parts of OpenMS (TOPPView&Co). This requires QtGui." ON)
 option(NO_WEBENGINE_WIDGETS "Do not use QtWebengineWidgets. Disables Javascript views in TOPPView." OFF)
+option(ENABLE_TORCH "Enable pyTorch" OFF)
```

Comment:
not sure where the best location is to query ENABLE_TORCH.

Reply:
Place looks fine but maybe call it WITH_OPENMS_ML? Or ENABLE_OPENMS_ML.

"Enabling torch" sounds kind of weird.